sdk/modelsrepository/Azure.Iot.ModelsRepository/samples/ModelsRepositoryClientSamples/ModelResolutionSamples.cs

```diff
@@ -17,26 +17,37 @@ internal class ModelResolutionSamples
 
         public static void ClientInitialization()
         {
-            // When no Uri is provided for instantiation. The global Azure IoT Models Repository endpoint is used
-            // and the dependency model resolution option is set to TryFromExpanded.
-            var client = new ModelsRepositoryClient();
+            #region Snippet:IoTModelsRepositorySamplesCreateServiceClientSimpleWithGlobalEndpoint
+
+            // When no Uri is provided for instantiation, the Azure IoT Models Repository global endpoint
```

Comment:
URI as an acronym should be in all uppercase

Reply:
The intent was to use the same casing as the dotnet type because it is in a dotnet code sample. But if you don't like it I will change it.

Reply:
Changed.

Reply:
If you are referring to the type, then that makes sense. In his case, to me at least, it seems like you are just speaking about the URI in a generic sense.